core/src/main/java/com/google/zxing/oned/Code39Writer.java

```diff
@@ -86,4 +96,60 @@ private static void toIntArray(int a, int[] toReturn) {
     }
   }
 
+  private static String tryToConvertToExtendedMode(String contents) {
+     int length = contents.length();
+     StringBuilder extendedContent = new StringBuilder();
+     for (int i = 0; i < length; i++) {
+       char character = contents.charAt(i);
+       switch (character) {
+         case '\u0000':
```

Comment:
I think this can be '\0' for consistency but don't change unless something else needs to change